packages/kotti-ui/source/kotti-form/controllers/KtFormControllerObject.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-	<div><slot /></div>
+	<div>
+		<slot
+			:setValue="(key, newValue) => setValue(key, newValue)"
```

Comment:
Why isn’t this `setValues`?

Reply:
sets a key only

Reply:
Yes, but why isn’t `setValues` exposed? It just makes sense to have `x` and `setX` exposed